exchanges/stream/websocket.go

```diff
@@ -347,34 +347,24 @@ func (w *Websocket) dataMonitor() {
 	w.Wg.Add(1)
 
 	go func() {
-		defer func() {
-			for {
-				// Bleeds data from the websocket connection if needed
-				select {
-				case <-w.DataHandler:
-				default:
-					w.setDataMonitorRunning(false)
-					w.Wg.Done()
-					return
-				}
-			}
-		}()
-
+		defer w.setDataMonitorRunning(false)
+		defer w.Wg.Done()
+		full := false
 		for {
 			select {
 			case <-w.ShutdownC:
 				return
 			case d := <-w.DataHandler:
 				select {
 				case w.ToRoutine <- d:
-				case <-w.ShutdownC:
-					return
+					if full {
+						full = false
```

Comment:
Part of me desires knowing just how many messages are not being processed due blockage. `verbose` does let me see ws messages, but this does not let me know when processing is back on. Would you consider a `if verbose` log to say its back?

Reply:
I did consider it, yes!
My issue with that is not wanting to do more processing than necessary for every single message going through.

What I had instead was a dropped counter, and a verbose check about it coming back.

Fixed 34bf736d

Note: Haven't had time to test this _at all_